.idea/gradle.xml

```diff
@@ -5,7 +5,7 @@
       <GradleProjectSettings>
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="1.8" />
```

Comment:
I suppose that this change is because you're using Java 7 instead of the last version, right?